### PR TITLE
chore: support for Oracle Linux 8

### PR DESCRIPTION
**Pull request by mdelapenya**
## What does this PR do?
It adds support for Oracle Linux 8 for the "running-on-beats" scenarios.

## Why is it important?
Increase the support matrix

## Checklist

- [ ] My code follows the style guidelines of this project
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have made corresponding change to the default configuration files
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have run the Unit tests (`make unit-test`), and they are passing locally
- [ ] I have run the End-2-End tests for the suite I'm working on, and they are passing locally
- [ ] I have noticed new Go dependencies (run `make notice` in the proper directory)

## Author's Checklist

- [ ] @jlind23 could you add the backport labels please? I do not see Oracle Linux in the support matrix spreadsheet so I'm not sure which branches should be used

## Related issues

- Closes https://github.com/elastic/e2e-testing/issues/2196

### CODE REVIEW

File: .ci/.e2e-tests-for-elastic-agent.yaml [hunk 1]
```diff
@@ -20,6 +20,10 @@ PLATFORMS:
     image: "ami-0d90bed76900e679a"
     instance_type: "c5.4xlarge"
     username: admin
+  oracle_linux8:
+    image: "ami-00371eeb8fd8e0e16"
+    instance_type: "c5.2xlarge"
```
Comment (mdelapenya) [inline .ci/.e2e-tests-for-elastic-agent.yaml:25]: Trying with a smaller instance 🤞

**Review by v1v — APPROVED** [review]
+1

Unrelated, could `.ci/.e2e-tests.yaml` and `.ci/.e2e-tests-for-elastic-agent.yaml` be renamed with something about daily/nightly? IIUC, this change will affect only the nightly builds, but the name of those files are not representing when they are used, just thinking if it could be helpful

**Comment by mdelapenya** [comment]
> +1
> 
> Unrelated, could `.ci/.e2e-tests.yaml` and `.ci/.e2e-tests-for-elastic-agent.yaml` be renamed with something about daily/nightly? IIUC, this change will affect only the nightly builds, but the name of those files are not representing when they are used, just thinking if it could be helpful

Indeed. I see those files as the metadata driving the execution (connecting VM settings, test suites, cucumber tags and jenkins parallel stages), so names that come to my mind are:

- `.e2e-tests.yaml` > `.stages-settings.yml`
- `.e2e-tests-for-elastic-agent.yaml` > `.stages-settings-elastic-agent.yml` (as it's triggered by the elastic-agent and beats repositories)

File: .ci/ansible/tasks/install_deps.yml [hunk 2]
```diff
@@ -49,6 +49,28 @@
     state: latest
   when: ansible_distribution in ["Fedora", "RedHat", "CentOS"]
 
+- name: Install dependencies (Oracle Linux)
```
Comment (mdelapenya) [inline .ci/ansible/tasks/install_deps.yml:52]: Same deps as for Centos/RedHat, but I wanted to keep them separated for decoupling.

If you think we can merge them, I'm OK with that too

**Review by jlind23 — APPROVED** [review]
Thanks Manu!

**Comment by mdelapenya** [comment]
The only failure is a well-know friend @ChrsMark is debugging. I'd say this is good to go.

@jlind23 could you confirm the target branches for this change?

**Comment by jlind23** [comment]
It should begin in 8.2 and then run on each version from 8.2 on.

**Pull request merged**